test/cmdlineTests/standard_function_debug_info/input.json

```
@@ -0,0 +1,19 @@
+{
+ "language": "Solidity",
+ "sources": {
+  "a.sol": {
+   "content": "// SPDX-License-Identifier: GPL-3.0\npragma abicoder v2; contract A { function f(uint[] memory) public view returns (uint256) { } }"
+  }
+ },
+ "settings": {
+  "evmVersion": "petersburg",
+  "outputSelection": {
+   "*": {
+    "A": [
+     "evm.deployedBytecode.funtionDebugData",
+     "evm.bytecode.functionDebugData"
+    ]
+   }
+  }
+ }
+}
```

Single-space indents really hurt to look at :P
```suggestion
{
    "language": "Solidity",
    "sources": {
        "a.sol": {
            "content": "// SPDX-License-Identifier: GPL-3.0\npragma abicoder v2; contract A { function f(uint[] memory) public view returns (uint256) { } }"
        }
    },
    "settings": {
        "evmVersion": "petersburg",
        "outputSelection": {
            "*": {
                "A": [
                    "evm.deployedBytecode.funtionDebugData",
                    "evm.bytecode.functionDebugData"
                ]
            }
        }
    }
}

```